### ISSUE TITLE
Key `p_village_135` repeated in `module_parties`

### ISSUE THREAD

**Issue by LordGolias** [LordGolias/Brytenwalda#4]
The item in [this line](https://github.com/LordGolias/Brytenwalda/blob/master/source/module_parties.py#L362), has the same key as the one 2 lines below.

The old compiler would accept this behaviour but would incorrectly return always the first entry of the list, which can lead to strange behaviours. 

The new compiler detects that this is an error, and correctly replaces the first line by the second, guaranteeing the uniqueness of the key.

Looking at the file,
```
("village_132","Eifionydd", ...),
("village_135","Sliab_Culinn", ...),
("village_134","Mag_Mucceda",  ...),
("village_135","Cogwy",  ...),
```
it seems that it is a simple mistake: `village_135` should be `village_133`.
However, modifying this line and compiling causes the game to crash on creating a new game, during `map init`.

**Comment by LordGolias** [comment]
> However, modifying this line and compiling causes the game to crash on creating a new game, during map init.

This was an error on the compiler that was fixed in bd6dbab.

**Comment by LordGolias** [comment]
Fixed in e4da599.
